internal/provider/provider_test.go

```diff
@@ -276,6 +276,124 @@ func notContainsCheckFunc(unexpectedSubString string) testingResource.CheckResou
 	}
 }
 
+func TestProvider_ConfigurationFlows(t *testing.T) {
+	t.Parallel()
+	t.Run("error path - user password login with missing data", func(t *testing.T) {
+		rec, _ := setupVCR(t, "fixtures/provider.error_user_pwd")
+		defer stopQuietly(rec)
+
+		testingResource.Test(t, testingResource.TestCase{
+			IsUnitTest:               true,
+			ProtoV6ProviderFactories: getProviders(rec.GetDefaultClient()),
```

Comment:
We don't need http communication for these error path tests, i.e. you should omit the VCR calls and use getProviders(nil) here.

Reply:
If client is nil the test will run into a panic